src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -204,6 +212,9 @@ class WiFiManager
     static void DisconnectHandler(uint8_t * data);
     static void PostConnectivityStatusChange(ConnectivityChange changeType);
     static void SendRouterSolicitation(System::Layer * layer, void * param);
+    static void Recover(System::Layer * layer, void * param);
+    static void ResetRecoveryTime(System::Layer * layer, void * param);
+    System::Clock::Milliseconds32 GetNextRecoveryTime();
```

Comment:
I would add a piece of doc which briefly explains the implemented timing strategy (exponential increase with defined limit).

Reply:
In that file, or somewhere in the documentation?

Reply:
Just adding a comment to the function declaration would be enough.